recipe/install-core.sh

```diff
@@ -2,8 +2,7 @@ INSTALL_DIR="$PREFIX/share/gap"
 
```

Comment:
Shouldn't this file start with a shebang?

Reply:
It is called by conda-build using /bin/bash, but added a shebang anyway